ADDRESSING.md

```diff
@@ -20,16 +18,21 @@
 
 ## TL;DR
 
-Native URI can be used for addressing:
-- immutable content:  `ipfs://{cid}`
-- mutable content behind IPNS record signed with a specific libp2p key: `ipns://{libp2p-key}` 
-- mutable content behind human-readable DNSLink (DNS TXT record): `ipns://{fqdn}` 
+For immutable content use an IPFS URI (`ipfs://{cid}`).
+- [ipfs://bafkreigh2akiscaildcqabsyg3dfr6chu3fgpregiymsck7e7aqa4s52zy](ipfs://bafkreigh2akiscaildcqabsyg3dfr6chu3fgpregiymsck7e7aqa4s52zy)
+
+For mutable content use an IPNS DNSLink URI (`ipns://{fqdn}`).
```

Comment:
Not mentioning `{libp2p-key}` here is a problem – people will cur corners and hardcode `ipns://` validation as "oh, is this a valid FQDN".  At the very least we should:

```suggestion
For mutable content use an IPNS DNSLink URI (human-readable `ipns://{fqdn}` or cryptographic  `ipns://{libp2p-key}`).
```

Reply:
I was shooting for have the TL;DR make a clear recommendation, and letting the rest of the doc explain all the things. I'm not adovcating for removing ipns://{libp2p-key} from the doc, just streamlining the TL;DR.